docs/advanced/public-api.md

```diff
@@ -0,0 +1,126 @@
+**Basic auth user name** and a **Basic auth password is passed** is passed in an `Authorization: Basic`
+header joined by a single colon `:` and **Base64 encoded** .
+
+e.g: `Basic MDhkN2UxMzYtZjE2OS02MzUyLTk5NmQtMjVkMGNmNzQ4ODFjOk9yWkM4dHB3a1V0b256UkFGSlZDdG5kV0o5Qm5DS05SL0lINjMxSzJCeFE9`
+
+### Create Public API credentials
+
+1. Open your [Public API credentials management page](https://app.configcat.com/my-account/public-api-credentials) on the ConfigCat Dashboard
+2. Click on the _Add New Credentials_ button and enter a name to identify your credential
+3. Click on the _Generate_ button: your credential will generated and it will show in a popup window
+4. Save your basic auth username and password to your favorite key-vault.
+
+	> **IMPORTANT:** if you close the popup window you won’t be able to access your password, and will need to create a new one!
+
+5. Use username-password credentials with BASIC authentication in your HTTP requests.
+
+	> Keep your credentials secure: do not embed it directly in your code and do not share it.
+
+## Service endpoints
+**[Detailed list of available endpoints](https://api.configcat.com/docs)**
+
+### Media types
+| Media                                                                 | Content-type         |
+| --------------------------------------------------------------------- | -------------------- |
+| json                                                                  | application/json     |
+| json with  [HAL](https://tools.ietf.org/html/draft-kelly-json-hal-08) | application/hal+json |
+
+> The JSON Hypertext Application Language (HAL) is a standard which
+   establishes conventions for expressing hypermedia controls, such as
+   links, with JSON 
+
+### Public API clients
+You can create clients easily using [Swagger Codegen](https://github.com/swagger-api/swagger-codegen).
+
+#### Sample client
+[ConfigCat Public API client for Angular](https://github.com/configcat/ng-configcat-publicapi)
+
+## Rate limits and throttling
+Public API's responsibility is to manage your products / environments / settings. It has the ability to serve the config value in some special cases but you **must not use** it to evaluate a setting value in your production environment.
+
+### Public API rate limit policy
+* The rate limit for the one endpoint call is `20` per second and `500` per hour per account credential.
+* The rate limit for the one endpoint call is `20` per second per client's IP address.
+
+>**When you hit your limit all api calls response status will be `HTTP429`**
+
+## Examples
+### Get Products
+#### Endpoint
+```GET ~/v1/products```
+#### Request - cURL
+```
+auth_credentials_in_base64="dGVzdDp0ZXN0"
+
+curl -X GET \
+"https://api.configcat.com/v1/products" \
+-H "accept: text/plain" \
+-H "Authorization: Basic ${auth_credentials_in_base64}"
+```
+#### Response
+```
+[{
+	"productId": "00d553ae-832b-5532-d8ad-9b2ae6ddfa11",
+	"name": "cat@example.com"
+}]
+```
+### Update feature flag
```

Comment:
i think it is very important to notice that this is not a regular update endpoint...it use SDK-KEY